src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js

```diff
@@ -94,7 +94,8 @@
                             callback(data);
                         }.bind(this))
                         .fail(function(data) {
-                            app.sandbox.logger.log('data could not be loaded:', data);
+                            app.logger.log('data could not be loaded:', data);
```

Comment:
I have to check this in more detail... When i look at it this way the statement a few lines above should also not be working (`app.*sandbox*.sulu.userSettings`).